### PR TITLE
skip userns tests on non-fedora distributions for now

### PR DESCRIPTION
**Pull request by baude**
Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: .papr.sh [hunk 1]
```diff
@@ -121,11 +132,11 @@ fi
 # Run integration tests
 if [ $integrationtest -eq 1 ]; then
     make TAGS="${TAGS}" test-binaries
-    SKIP_USERNS=1 make varlink_generate GOPATH=/go
+    make varlink_generate GOPATH=/go
     if [ $runpython -eq 1 ]; then
-        SKIP_USERNS=1 make clientintegration GOPATH=/go
+        make clientintegration GOPATH=/go
```
Comment (TomSweeneyRedHat) [inline .papr.sh]: do you not need the `$INTEGRATION_TEST_ENVS` bit here and at line 135?

**Comment by TomSweeneyRedHat** [comment]
@baude some of the tests are a bit red.

**Comment by mheon** [comment]
They mostly seem to be flakes

**Comment by mheon** [comment]
bot, retest this please

File: .papr.sh (hunk 1 above)
Comment (baude) [reply .papr.sh]: no, the env variable is only for integration tests.

Comment (mheon) [reply .papr.sh]: Not bare metal - Travis runs in a container right now

**Pull request merged**